build/Dockerfile.aws-cli

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.22-bullseye AS builder
+ARG OS=linux
+ARG ARCH=amd64
+
+# Downloading aws-cli
+WORKDIR /tmp
+RUN apt-get update
+RUN apt-get install unzip
+RUN unzip -v
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
can we make this as arch agnostic? we would also need to support other arch.

Reply:
Pushed the change, please advise if it looks ok. Will fix the commit sign-off tomorrow moning.

Reply:
looks good